streaming/src/main/scala/org/apache/gearpump/streaming/executor/TaskLauncher.scala

```diff
@@ -40,12 +40,12 @@ class TaskLauncher(
     taskActorClass: Class[_ <: Actor])
   extends ITaskLauncher{
   def launch(taskIds: List[TaskId], argument: TaskArgument, context: ActorRefFactory): Map[TaskId, ActorRef] = {
-    import argument.{processorDescription, subscribers}
+    import argument.{appName, processorDescription, subscribers}
```

Comment:
appName then can be retrieved by query ExecutorContext.  No need to put it in TaskArgument.